examples/kubernetes/compiled/minikube-mysql/manifests/mysql_secret.yml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 data:
-  MYSQL_ROOT_PASSWORD: P3tncGc6bXlzcWwvcm9vdC9wYXNzd29yZH0=
+  MYSQL_ROOT_PASSWORD: ?{gpg:mysql/root/password:20632339}
```

Comment:
mind you, this requires users to have a fully gpg setup which might be a PITA. I'd leave it cleartext untile we implement the plain backend.

Reply:
https://github.com/deepmind/kapitan/pull/102

Just made another PR with updates to docs to include details about secrets. We should make the examples as realistic as possible. Also the GPG keys are already in `examples/kubernetes/secrets` so anyone can set them up.

Reply:
not 100% convinced. Ask @gburiola to bless :)
I'd prefer we reduce the chance of the Kapitan crashing on first use because the setup in incomplete. That would look poor.

Reply:
It still compiles though without gpg setup

Reply:
right.. I thought you had the randomstr example. That would fail.

Reply:
I like the idea of having a complete example including GPG. Maybe we could add a `README` file at `kapitan/examples/kubernetes` saying that the example uses GPG encrypted secrets. It should compile but in order to apply user would need to install GPG.

Another idea would be to have a `passphrase` encryption backend that asks for the `passphrase` at the command line when you run `kapitan secrets --reveal`

Reply:
Added docs in the other PR, will update them to include GPG setup with the example keys. Was thinking of doing the `aes256` backend as well but we have to refactor the current gpg setup.